### PR TITLE
Create Etherscan util to consolidate etherscan URL related code

### PR DESCRIPTION
**Pull request by mikedemarais**

### CODE REVIEW

File: src/hooks/useEtherscan.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { isString } from 'lodash';
+import { useCallback, useMemo } from 'react';
+import { Linking } from 'react-native';
+import useAccountSettings from './useAccountSettings';
+import { ethereumUtils } from '@rainbow-me/utils';
+
+export default function useEtherscan() {
+  const { network } = useAccountSettings();
+  const etherscanHost = useMemo(
```
Comment (osdnk) [inline src/hooks/useEtherscan.js:9]: Assuming it's just a string and it's cheap, I would suggest not using memorization here.

Therefore, I cannot see then the reason why `openTokenEtherscanURL` and `openTransactionEtherscanURL` won't be just static functions that can be provided as some helper?

Comment (jinchung) [reply src/hooks/useEtherscan.js:9]: Agreed, let's make this a util!

Comment (mikedemarais) [reply src/hooks/useEtherscan.js]: @jinchung @osdnk okie! made the change here -> https://github.com/rainbow-me/rainbow/pull/1552/commits/69ed637d61b01dd345d380799591443049963125

lemme know if u have further tweaks youd like me to make

File: src/utils/ethereumUtils.js [hunk 2]
```diff
@@ -8,15 +8,19 @@ import {
...
+} from 'lodash';
+import { Linking, NativeModules } from 'react-native';
 import { ETHERSCAN_API_KEY } from 'react-native-dotenv';
 import URL from 'url-parse';
-import {
-  DEFAULT_HD_PATH,
-  identifyWalletType,
-  WalletLibraryType,
-} from '../model/wallet';
+import { getNetwork } from '@rainbow-me/handlers/localstorage/globalSettings';
```
Comment (jinchung) [inline src/utils/ethereumUtils.js:23]: We can use network from redux store and not need to read from lcoalstorage every time for this! (And thereby get rid of await / async)

Comment (mikedemarais) [reply src/utils/ethereumUtils.js]: okie how about now https://github.com/rainbow-me/rainbow/pull/1552/commits/e2219f58f9a1b6ab94fa24cd4c66d9c0414fc784

**Review by jinchung — APPROVED** [review]
✨

**Review by skylarbarrera — APPROVED** [review]
👌

**Pull request merged**